wsapi.go

```diff
@@ -276,7 +276,7 @@ func (s *Session) UpdateStreamingStatus(idle int, game string, url string) (err
 		return ErrWSNotFound
```

Comment:
Move this block into UpdateStatusComplex, it should be thread safe too! :)

Reply:
Sorry, I think you missed this, we should move the RLock/wsConn check to UpdateStatusComplex.

Reply:
Yeah sorry I missed that.